src/main/scripts/prepare-release.py

```diff
@@ -50,16 +50,17 @@ def gen_changelog(repo_path):
 
     changelog = cli.log(f'{lasttag}..main', graph=True, pretty='format:%s', abbrev_commit=True, date='relative')
     changelog = changelog.split('\n')
-
-    return list(filter(changelog_filter, changelog))
+    filtered_changelog = list(filter(changelog_filter, changelog))
+    sanitized_changelog = map(lambda c: re.sub(r'CLIP-[0-9]{2,4}(?![0-9]): ', '', c), filtered_changelog)
```

Comment:
This won't match CLIP-12345 (ie 5 digits)

Reply:
We are at CLIP-17xx. We can come back to this in 5 years later